Update readme.md (#17)
UBC-MDS/pygtracker: README.md

## Package Overview
`pygtracker` is a Python package that allows UBC lecturers to record, analyze and adjust grades for students in a particular program. Users can record grades from each course, generate a summary report to determine which class is more challenging than the rest, or identify students who may need help. Finally, the package can suggest ways to adjust the grades for students to ensure the average grade matches a predefined benchmark. It contains five functions: `register_courses`, `record_grades`, `summarize_courses`, `rank_students` and `suggest_grade_adjustment`.

The main components of this package are:

> - Register the courses

Reviewer: ```suggestion
- Register courses
```

  - Read/store the courses information as a dataframe

> - Record the grade for students

Reviewer: ```suggestion
- Record grades for students
```

>   - Read/store the students grade for each assessment as a data frame

Reviewer: ```suggestion
  - Read/store the students' grades for each assessment as a data frame
```

> - Summarize the grades by course

Reviewer: ```suggestion
- Summarize grades by courses
```

  - Provide summary statistics on the courses 
  - Provide the ranking of courses based on courses' average grade

> - Summarise the grades by students

Reviewer: ```suggestion
- Summarise grades by students
```

  - Provide the ranking of students' average grades for the course selected (or the whole program).
- Suggest grade adjustment

>   - Suggest grade adjustments for any course based on a predefined benchmark

Reviewer: ```suggestion
  - Suggest grade adjustments for any course based on predefined benchmarks
```
(We have 3 benchmarks: course, lab, quiz)
